src/utilities/alignment/run_10x_vdj.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python
+
+# Example: TAXON=homo CELL_COUNT=3000 S3_DIR=s3://biohub-spyros/data/10X_data/CK_Healthy/ ./10x_count.py
+import argparse
+import os
+import pathlib
+import sys
+import subprocess
+import tarfile
+import posixpath
+
+from utilities.log_util import get_logger, log_command
+
+import boto3
+
+
+# reference genome bucket name for different regions
+S3_REFERENCE = {"east": "czbiohub-reference-east", "west": "czbiohub-reference"}
+
+# valid and deprecated reference genomes
+reference_genomes = {
+    "homo": "HG38-PLUS",
+    "hg38-plus": "HG38-PLUS",
+    "homo.gencode.v30.ERCC.chrM": "homo.gencode.v30.annotation.ERCC92",
+    "mus": "MM10-PLUS",
+    "mm10-plus": "MM10-PLUS",
+    "mm10-1.2.0": "mm10-1.2.0",
+    "mus-premrna": "mm10-1.2.0-premrna",
+    "mm10-1.2.0-premrna": "mm10-1.2.0-premrna",
+    "hg19-mm10-3.0.0": "hg19-mm10-3.0.0",
+    "microcebus": "MicMur3-PLUS",
+    "gencode.vM19": "gencode.vM19",
+    "GRCh38_premrna": "GRCh38_premrna",
+    "zebrafish-plus": "danio_rerio_plus_STAR2.6.1d",
+    "GRCh38-VDJ":"refdata-cellranger-vdj-GRCh38-alts-ensembl-4.0.0",
+}
+deprecated = {
+    "homo": "hg38-plus",
+    "mus": "mm10-plus",
+    "mus-premrna": "mm10-1.2.0-premrna",
+}
+
+# other helpful constants
+CELLRANGER = "cellranger"
+S3_RETRY = 5
+
+
+def get_default_requirements():
+    return argparse.Namespace(
+        vcpus=64, memory=256000, storage=2000, ecr_image="demuxer"
+    )
+
+
+def get_parser():
+    """ Construct and return the ArgumentParser object that parses input command.
+    """
+
+    parser = argparse.ArgumentParser(
+        prog="run_10x_count.py",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description="Run alignment jobs using 10x",
```

Comment:
change this to specify that it's a VDJ alignment

Reply:
good idea